core-java-modules/core-java-lang-oop-patterns/src/test/java/com/baeldung/returnfirstnonnull/ReturnFirstNonNullUnitTest.java

```diff
@@ -0,0 +1,108 @@
+package returnfirstnonull;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import org.apache.commons.lang3.ObjectUtils;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Predicates;
+import com.google.common.collect.Iterables;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class ReturnFirstNonNullUnitTest {
+
+    private List<Object> objects;
```

Comment:
As in the other tests, I think string examples would be clearer than raw objects

Reply:
Changed as suggested